### PR TITLE
Update client.dart

### PR DESCRIPTION
**Pull request by oblivioned**
Add "value" field in using RPC interface "eth_call".

### CODE REVIEW

File: lib/src/core/client.dart [hunk 1]
```diff
@@ -375,10 +375,12 @@ class Web3Client {
       {EthereumAddress sender,
       @required EthereumAddress contract,
       @required Uint8List data,
+      EtherAmount value,
       BlockNum atBlock}) {
     final call = {
       'to': contract.hex,
       'data': bytesToHex(data, include0x: true, padToEvenLength: true),
+      'value': bytesToHex(value.getInWei.toRadixString(16), include0x: true),
```
Comment (simolus3) [inline lib/src/core/client.dart:383]: Ah wait, I think this needs to be
```suggestion
      'value': '0x${value.getInWei.toRadixString(16)}',
```

Comment (oblivioned) [reply lib/src/core/client.dart:383]: You are right. 

Do we need to add the "value" field in "estimateGas" and "call" ?

Comment (simolus3) [reply lib/src/core/client.dart:383]: I think it makes sense to add them there as well, yes :+1:

Comment (oblivioned) [reply lib/src/core/client.dart:383]: OK, let me do it.

**Pull request closed without merging**